src/test/java/teammates/ui/webapi/DeleteDeadlineExtensionActionTest.java

```diff
@@ -0,0 +1,103 @@
+package teammates.ui.webapi;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.DeadlineExtensionAttributes;
+import teammates.common.util.Const;
+import teammates.ui.output.MessageOutput;
+
+/**
+ * SUT: {@link DeleteAccountRequestAction}.
+ */
+public class DeleteDeadlineExtensionActionTest extends BaseActionTest<DeleteDeadlineExtensionAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.DEADLINE_EXTENSION;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return DELETE;
+    }
+
+    @Override
+    @Test
+    protected void testExecute() {
+        DeadlineExtensionAttributes deadlineExtension = typicalBundle.deadlineExtensions.get("student4InCourse1Session1");
+
+        ______TS("Not enough parameters");
+
+        verifyHttpParameterFailure();
+
+        ______TS("Missing parameter");
+
+        String[] params = new String[] {
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, deadlineExtension.getFeedbackSessionName(),
+                Const.ParamsNames.USER_EMAIL, deadlineExtension.getUserEmail(),
+                Const.ParamsNames.IS_INSTRUCTOR, Boolean.toString(deadlineExtension.getIsInstructor()),
+        };
+
+        verifyHttpParameterFailure(params);
+
+        params = new String[] {
+                Const.ParamsNames.COURSE_ID, deadlineExtension.getCourseId(),
+                Const.ParamsNames.USER_EMAIL, deadlineExtension.getUserEmail(),
+                Const.ParamsNames.IS_INSTRUCTOR, Boolean.toString(deadlineExtension.getIsInstructor()),
+        };
+
+        verifyHttpParameterFailure(params);
+
+        params = new String[] {
+                Const.ParamsNames.COURSE_ID, deadlineExtension.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, deadlineExtension.getFeedbackSessionName(),
+                Const.ParamsNames.IS_INSTRUCTOR, Boolean.toString(deadlineExtension.getIsInstructor()),
+        };
+
+        verifyHttpParameterFailure(params);
+
+        params = new String[] {
+                Const.ParamsNames.COURSE_ID, deadlineExtension.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, deadlineExtension.getFeedbackSessionName(),
+                Const.ParamsNames.USER_EMAIL, deadlineExtension.getUserEmail(),
+        };
```

Comment:
Would it be good to use commented lines to indicate which parameter is missing?
`// Const.ParamsNames.USER_EMAIL`